develop/smart-contracts/overview.md

```diff
@@ -1,4 +1,145 @@
 ---
 title: Overview
-description: TODO
----
\ No newline at end of file
+description: Learn how developers can build smart contracts on Polkadot by leveraging either WASM/ink! or EVM contracts across many different parachains.
+---
+
+# Overview
+
+## Introduction
+
+Polkadot is a chain specifically designed to service other chains. As a result, developers cannot deploy smart contract applications directly to Polkadot.
+
+Therefore, developers looking to build smart contract applications in Polkadot need to leverage the smart contract functionality that chains in the Ecosystem offer. Developers can generally choose between two main development paths within the ecosystem. 
+
+On one end, developers can leverage WASM-based smart contracts chains that support [ink!](TODO:update-path){target=\_blank}, a Rust-based smart contract language that acts as an embedded domain-specific language (eSDL) for Rust. This allows developers to benefit from Rust’s syntax, type safety, memory safety, and extensive tooling.
```

Comment:
```suggestion
On one end, developers can leverage Wasm-based smart contracts chains that support [ink!](TODO:update-path){target=\_blank}, a Rust-based smart contract language that acts as an embedded domain-specific language (eDSL) for Rust. This allows developers to benefit from Rust's syntax, type safety, memory safety, and extensive tooling.
```

Reply:
`TODO:update-path` is a flag for internal links. In this context, we can add this reference to ink! - https://use.ink/